test/library/packages/LinearAlgebra/correctness/testEigen.chpl

```diff
@@ -0,0 +1,120 @@
+use LinearAlgebra, Random;
```

Comment:
Were these tests derived from another test suite?

Reply:
No, I wrote them from scratch.  The numbers used in the matrix in testCholesky were taken from https://rosettacode.org/wiki/Cholesky_decomposition to make sure I ended up with something reasonable.